packages/generated/scripts/make-config.sh

```diff
@@ -1,6 +1,8 @@
-#!/bin/bash
+#!/bin/sh
 set -euo pipefail
-cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+cd "$(dirname "$0")"
+
 cd ..
 
 node ./scripts/make-config.js
```

Comment:
@sergekh2 i needed to add these changes to make this script work on alpine. please let me know if i'm missing anything here

Reply:
edit: I'm deleting this whole file. seems like the only purpose it serves is to cd into `packages/generated` before running `node ./scripts/make-config.js`. the same outcome can be achieved by running the node script directly from `package.json`. i'm also updating the `scripts/deploy-contracts.sh` script to run this new package.json script